scripts/install.sh

```diff
@@ -168,6 +170,11 @@ function outro() {
         dockerRegistryIP=" docker-registry-ip=$DOCKER_REGISTRY_IP"
     fi
 
+    local noProxyAddrs=""
+    if [ -n "$PROXY_ADDRESS" ]; then
+        noProxyAddrs=" additional-no-proxy-addresses=${SERVICE_CIDR},${POD_CIDR}"
```

Comment:
The join script has the yaml spec so it already has any additionalNoProxyAddresses defined in there but it doesn't know what CIDRs are in use. These will be appended to anything defined in the spec.